### ISSUE TITLE
[Bug report]: 

### ISSUE THREAD

**Issue by jayscee** [iotaledger/firefly#4805]
### Platform

Desktop

### Version

1.6.5

### Description

I have i firefly a total of 1934 SMR in 2 wallets (wallet 1: 1738; wallet 2: 198) after staking for full period.   
After trying many times Shimmer has only found 1082 SMR, leaving a shortfalll of 852 SMR.

Judging by number of other IOTA stakers that have not received the correct SMR total, I assume there is a bug in shimmer that I assume you are aware of and are trying to fix.

There is also nowhere on shimmer site for me to seek the missing SMRs that are still visible on firefly.

### Expected behaviour

To find total of 1934 SMRs.

### Actual behaviour

Only 1082 SMR found after amny, many attempts

### Can the issue reliably be reproduced?

No

### Operating System

Windows

### Developer Profile

No

### Profile Type

Software (Stronghold)

### Crash Status

None of the above

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nicole-obrien** [comment]
Have you tried claiming your shimmer using the Firefly Shimmer wallet?

**Comment by jayscee** [comment]
Yes I claimed the 1082 SMR it found, but now there is nowhere in shimmer to
search for the missing amount.

**Comment by begonaalvarezd** [comment]
Hi, is this still a problem? Did you manage to claim all your SMR?

**Comment by begonaalvarezd** [comment]
@jayscee we recently released a new version fixing multiple issues, shimmer claiming related too, could you update to the latest version and let us know if the issue is fixed for you 🙏🏼 ?

**Comment by begonaalvarezd** [comment]
If you found more rewards you can keep searching to find what you think you are missing, its a bit of a pacience game, it all depends on the addresses you used to stake, if you staked with addresses with very high indexes your rewards will take some time to get found
